src/stan-users-guide/simple_linear_regression/regression_efficient.stan

```diff
@@ -0,0 +1,19 @@
+data {
+  int<lower=0> N;
+  vector[N] x;
+  vector[N] y;
+}
+
+parameters {
+  real alpha_hat;
+  real beta_hat;
+  real<lower=0> sigma_hat;
+}
+
+//Just vectorized right now. 
+model {
+  alpha_hat ~ normal(5,10);
```

Comment:
Spaces after commas always.

Reply:
Is there a stan linter? I started to write one just to catch this stuff but before I go any further I thought I'd ask.